packages/phasor/src/elements/brush/brush-element.ts

```diff
@@ -0,0 +1,76 @@
+import { Utils } from '../../utils/tl-utils.js';
+import { Vec } from '../../utils/vec.js';
+import { deserializeXYWH, serializeXYWH } from '../../utils/xywh.js';
+import { BaseElement, HitTestOptions } from '../base-element.js';
+
+export class BrushElement extends BaseElement {
+  type = 'brush' as const;
+  color = '#000000';
+  points: number[][] = [];
```

Comment:
Is it possible to use `number[]` with the `[x1, y1, x2, y2...]` layout that could reduce the serialized size?

Reply:
I'm not sure about this change. Because if we want to add brush pressure in the future, it's not easy for upgrading.

Reply:
Okay so let's keep using nested array for `pressure`.